### PR TITLE
Makefile changes, README.md, removes FPGA code

### PR DESCRIPTION
**Pull request by pkubanek**
Improves Makefile. Running make VERBOSE=1 will provide verbose output. Added README.md with basics. Remove LabView FPGA code, which is now stored in separate project (ts_m1m3supportFPGA).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+primary when it's raised on controlled lowered. Hardpoints are also used to
+make mirror stiff, removing wind induced oscillations.
+
+There are secondary goals operating DAAs and SAAs:
+
+* offload local hardpoints steering forces
+* apply small forces to keep mirror in optically ideal shape
+
+DAAs and SAAs are also equipped with booster valves. Those are activated to
+counteract forces experiented during telescope slews.
```
Comment (awclemen) [inline README.md]: experienced ?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+coils in passive state. Any failure or detected problem (seismic event,..)
+shall put mirror quickly to lowered position. Of course the controller and cell
+hardware shall not drop the mirror into a free fall. Emergency lowering of the
+mirror is referred as cell panic in the following text.
+
+DAA and SAA are equipped with force cells. Difference between target and
+measured force is measured and excessive values leads to cell panic.
+
+Hardpoints are equipped with pneumatic driven break-away mechanism. Any
+excessive force will break hardpoint, and once detected with control software,
```
Comment (awclemen) [inline README.md]: make the hardpoint break away

File: README.md (hunk 1 above)
Comment (pkubanek) [reply README.md]: Fixed. Plus supp**p**ort and explain ILC

File: README.md (hunk 2 above)
Comment (pkubanek) [reply README.md]: Fixed. Thanks.

**Pull request merged**